content/1.7/back-office/shipping/carriers/carriers-listing.md

```diff
@@ -0,0 +1,107 @@
+ 
+  - Add new carrier
+
+👉  See the [specs of the add/edit carrier]({{<ref "add-edit-carrier.md">}} "Add/edit carrier") 
+
+## Settings wheel
+
+When clicking on the settings wheel, the following options are displayed:
+
+1. Refresh list
+
+After clicking on it, the page is reloaded. Sorts and searches are kept.
+
+2. Show SQL query
+
+After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.
+
+There are two buttons:
+
+- Close: After clicking on it, the modal is closed
+- Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+3. Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+## Columns
+
+The page displays the carriers list with the following columns:
+
+1. ID
+
+It's possible to sort by id asc or desc and search by id.
+
+2. Name
+
+It's possible to sort by name of carriers asc or desc and search by name.
+
+3. Logo
+
+The logo of the carrier is displayed if there is any
+
+4. Delay
+
+The delay of the shipping method is displayed. It's possible to search by delay.
+
+5. Status
+
+The carrier can be enabled or disabled. If it's enabled, a green checkmark is displayed. If it's disabled, a red cross is displayed. It's possible to search by status by clicking on the dropdown and by selecting one of the two options: "yes" or "no". If "yes" is selected, only the carriers enabled are displayed. If "no" is selected, only the disabled carriers are displayed.
+
+6. Free shipping
+
+Status if the carrier provides free shipping. If active - green check mark. If inactive - red cross
+
+7. Position
+
+The position of each carrier can be changed by clicking on this column and by moving it up or down.
+After changing the position, the message "Successful update." is displayed. It's possible to sort by position asc or desc and search by position.
+
+8. Actions
+
+- **Edit.** This is the main action and it can be performed by clicking on the line or on the button. 
+ 👉  See the [specs of the add/edit carrier]({{<ref "add-edit-carrier.md">}} "Add/edit carrier") 
+
+- **Delete.** The carrier can be deleted by clicking on this button. After clicking on "Delete", a modal with the title "Delete selection" is opened asking to confirm or cancel the  action: "Are you sure you want to delete the selected item(s)?"
+ The action can be canceled by clicking on the cross or on the cancel button.
+ After clicking on "Delete" button, the carrier is deleted and a successful message is displayed "Successful deletion."
+ 
+ ### Search
+ 
+The search button is disabled as long as there are less than 2 carriers in the list. 
+
+A search can be done by ID, name, delay, status (drop-down menu), free shipping (drop-down) and position.
+It's possible to sort the list by carriers or by search in an ascending or descending way.
```

Comment:
```suggestion

```

Reply:
I have the impression that this sentence is a duplicate, it is already explained in the "columns" part